browser/js/dashboard/dashboard.controller.js

```diff
@@ -1,17 +1,18 @@
 
-app.controller('dashboardCtrl', ['$scope', 'DashboardFactory', '$rootScope', 'dashList',
-    function($scope, DashboardFactory, $rootScope, dashList) {
+app.controller('dashboardCtrl', ['$stateParams', '$scope', 'DashboardFactory', '$rootScope', 'dashList',
+    function($stateParams, $scope, DashboardFactory, $rootScope, dashList) {
 
     $scope.editable = false;
-    $scope.dashName = "You have no dashboards";
-    $scope.dashDesc = "Please create a dashboard";
+    $scope.dashName = $stateParams.name || "You have no dashboards";
```

Comment:
I can't find "dashName" anywhere else in the codebase besides this file. Where is it being used?